Resources/Prototypes/_White/Shaders/shaders.yml

```
@@ -0,0 +1,4 @@
+- type: shader
+  id: NightVision
+  kind: source
+  path: "/Textures/_White/Shaders/nightvision.swsl"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)